pkg/geo/encode.go

```diff
@@ -311,3 +312,57 @@ func StringToByteOrder(s string) binary.ByteOrder {
 		return DefaultEWKBEncodingFormat
 	}
 }
+
+// LineFromEncodedPolyline takes the encoded polyline ASCII and precision, decodes the points and returns them as a geometry
+func LineFromEncodedPolyline(encodedPolyline string, precision int) (Geometry, error) {
```

Comment:
Two things:
* Can you move the polyline logic (decodePolylinePoints, decodePointValue) to polyline.go.
* Can you rename this to ParseEncodedPolyline, and move this to parse.go?

Reply:
Ok, done